types/src/transaction/mod.rs

```diff
@@ -235,6 +235,17 @@ impl RawTransaction {
         )))
     }
 
+    pub fn multi_sign_for_testing(
```

Comment:
shouldn't this be exposed only for 

#[cfg(any(test, feature = "fuzzing"))]

or maybe even inline where you are using it?

Reply:
Yes, thanks. Good idea!